src/toast.rs

```diff
@@ -168,6 +172,12 @@ impl Toast {
         self
     }
 
+    /// Can show progress bar?
```

Comment:
I think better wording should be something like `Should a progress bar be shown?`